AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -123,26 +93,18 @@ msgid "Registrados"
 msgstr ""
 
 #: AlertaDengue/dados/charts/home.py:47
-#, fuzzy, python-brace-format
-#| msgid "%{x} <br>Semana: %{text} <br>%{y} Casos Estimados<extra></extra>"
 msgid "<br>SE %{text}<br>%{y:1f} Casos notificados<extra></extra>"
 msgstr "%{x} <br>Week: %{text} <br>%{y} Estimated Cases<extra></extra>"
 
 #: AlertaDengue/dados/charts/home.py:58
-#, fuzzy
-#| msgid "Casos estimados"
 msgid "Estimados"
 msgstr "Estimated cases"
 
 #: AlertaDengue/dados/charts/home.py:63
-#, fuzzy, python-brace-format
-#| msgid "%{x} <br>Semana: %{text} <br>%{y} Casos Estimados<extra></extra>"
 msgid "<br>SE %{text}<br>%{y:1f} Casos estimados<extra></extra>"
 msgstr "%{x} <br>Week: %{text} <br>%{y} Estimated Cases<extra></extra>"
 
 #: AlertaDengue/dados/charts/home.py:75
-#, fuzzy
-#| msgid "Número acumulado de casos no ano"
 msgid "Total de casos no estado"
 msgstr "Accumulated number of cases per year"
```

Comment:
It seems that this refers to the "cumulative number of cases in the state"

Reply:
I recommend doing the translations for each page in different pull requests.